engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToHMDNode.java

```diff
@@ -117,5 +119,14 @@ private void renderFinalStereoImage(RenderingStage renderingStage) {
                 GL11.glFinish();
                 break;
         }
+
+        // Bind the default FBO. The DAG does not recognize that this node has
+        // bound a different FBO, so as far as it is concerned, FBO 0 is still
+        // bound. As a result, without the below line, the image is only copied
+        // to the HMD - not to the screen as we would like. To get around this,
+        // we bind the default FBO here at the end.  This is a bit brittle
+        // because it assumes that FBO 0 is bound before this node is run.
+        // TODO: break this node into two different nodes that use addDesiredStateChange(BindFBO...))
+        new BindFBOTask(0,DEFAULT_FRAME_BUFFER_URN).execute();
```

Comment:
Eeeeeew. :stuck_out_tongue: 

Yes, this is pretty ugly. :smile: 

There is no advantage in using the BindFBOTask here - which, by the way, might become a private class as is not intended for direct instantiation. I therefore recommend for the time being simply using

`glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, 0)`

and then, as you suggest, breaking this node into two nodes, each binding the appropriate FBO.

By the way, are you sure you need to clear the left/rightEye buffers? And if yes, do they even have a depth attachment?